_attributes/common-attributes.adoc

```diff
@@ -67,8 +67,8 @@ endif::openshift-origin[]
 :mtc-version: 1.7
 :mtc-version-z: 1.7.10
 // builds (Valid only in 4.11 and later)
-:builds-v2title: Builds for Red Hat OpenShift
-:builds-v2shortname: OpenShift Builds v2
+:builds-v2title: Red Hat OpenShift Builds  
+:builds-v2shortname: OpenShift Builds
 :builds-v1shortname: OpenShift Builds v1
```

Comment:
It should be differecianted from Openshift Builds. May be Builds classic.  As we decided to not add v2 to  Openshift builds

Reply:
@Preeticp what name would you suggest for the `builds-v1shortname` attribute? Would it be Classic Builds or Core Builds?